### DISCUSSION TITLE
Custom CartesianGrid

### DISCUSSION THREAD

**Discussion by paur94** [recharts/recharts#4183] [General]
Hi,
Can I create a custom CartesianGrid component in Recharts.js and change the color of a specific vertical line? For example color red line in this image.
![image](https://github.com/recharts/recharts/assets/60259252/fd859317-41de-406d-9c39-a26f9efb9bb1)
Thank you,

**Comment by ckifer** [comment]
Hi,

You can change the color of all of the vertical lines or you can create a reference line, but you can't change the color of a single grid line. It looks like ReferenceLine might be what you want though
